### PR TITLE
Add sigs for common Rails ::Dirty API 

### PR DESCRIPTION
**Pull request by manhhung741**
See 
 https://www.rubydoc.info/gems/activerecord/ActiveRecord/AttributeMethods/Dirty#saved_change_to_attribute%3F-instance_method

And

https://www.rubydoc.info/gems/activemodel/ActiveModel/Dirty

### CODE REVIEW

File: lib/activerecord/all/activerecord.rbi [hunk 1]
```diff
@@ -9,3 +9,9 @@ class ActiveRecord::Migration::Current < ActiveRecord::Migration
   def change_column(table_name, column_name, type, options = nil); end
   def create_table(table_name, options = nil); end
 end
+
+module ActiveRecord::AttributeMethods::Dirty
+  extend T::Sig
+  sig { params(attr_name: Symbol, options: T.untyped).returns(T::Boolean) }
+  def saved_change_to_attribute?(attr_name, **options); end
```
Comment (manhhung741) [inline lib/activerecord/all/activerecord.rbi]: This matches the API generated by sorbet in gems/ folder

  def saved_change_to_attribute?(attr_name, **options); end

File: lib/activemodel/all/activemodel.rbi [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# typed: strong
+
+module ActiveModel::Dirty
+  extend T::Sig
+  sig { params(attr: Symbol, from: T.untyped, to: T.untyped).returns(T::Boolean) }
```
Comment (manhhung741) [inline lib/activemodel/all/activemodel.rbi]: Also match API generated by sorbet in gems/ folder

**Comment by DarkDimius** [comment]
Thank you, Harry

**Comment by manhhung741** [comment]
Uhh, the signature for `attribute_changed?` is different on Rails v4.2. I'll try to put another PR for Rails 4.2.

On 5.0+
```
  def attribute_changed?(attr, from: nil, to: nil); end
```
On 4.2
```
  def attribute_changed?(attr, options = nil); end
```

**Pull request merged**